my-app/src/components/episodes/EpisodeList.js

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from "react"
+import EpisodeCard from "./EpisodeCard"
+import axios from "axios"
+
+const EpisodeList = () => {
+  const [EpisodeList, setEpisodeList] = useState([])
+  useEffect(() => {
+    axios.get("https://rickandmortyapi.com/api/episode").then((response) => {
+      setEpisodeList(response.data.results)
+    })
+  }, [])
+
+  return (
+    <div>
+      {EpisodeList.map((episode, id) => {
```

Comment:
it's better to name second attribute inside of the map as `index` instead of `id`.
Because it's not really an id. it's just a number from 0 to the total items in the array

```suggestion
{EpisodeList.map((episode, index) => {
```